filters/builtin/stripquery_test.go

```diff
@@ -91,3 +92,21 @@ func TestPreserveQuery(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkStripQuery(b *testing.B) {
```

Comment:
Rename to BenchmarkSanitize OR rewrite to benchmark the filter, see e.g. https://github.com/zalando/skipper/blob/37c474ad7ce52f529028cd8de8dcbe722df0e6ed/filters/builtin/header_test.go#L547-L559 (make sure each run gets the same input though)

Reply:
renamed to `BenchmarkSanitize`